src/databricks/labs/ucx/source_code/known.json

```diff
@@ -3141,6 +3141,83 @@
     "dockerpycreds.utils": [],
     "dockerpycreds.version": []
   },
+  "duckdb": {
+    "adbc_driver_duckdb": [],
+    "adbc_driver_duckdb.dbapi": [],
+    "duckdb": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate 'conn.table(table_name, **kwargs)' because its table name argument cannot be computed"
```

Comment:
I expect that this is a false positive. It is probably a duckdb connection, not a Spark connection

Reply:
fixed